### ISSUE TITLE
Can this detector return the angle from 0 to 2*Pi radians?

### ISSUE THREAD

**Issue by khiemauto** [csuhan/s2anet#111]
I ran demo/demo_inference_without_dataset.py with a picture of many planes with different angles but it seems that the angle result only returns between [-pi/4, 3*pi/4] radians.

Can this detector return the angle from [0, 2*pi] radians (0-360 degree) with custom dataset?

**Comment by csuhan** [comment]
You can return [0, 2pi] by changing here:
https://github.com/csuhan/s2anet/blob/7d66620f02b29e9008fbdb91cfcb10916ea47c49/mmdet/core/bbox/transforms_rotated.py#L7
But it may take more training time to converge.

**Comment by khiemauto** [comment]
In config file, i check using BboxOverlaps2D_rotated
https://github.com/csuhan/s2anet/blob/7d66620f02b29e9008fbdb91cfcb10916ea47c49/configs/dota/s2anet_r50_fpn_1x_dota.py#L48-L56

I had test code
```python
from mmdet.core.bbox.iou_calculators.iou2d_calculator_rotated import BboxOverlaps2D_rotated
import torch, math
iou_rotate = BboxOverlaps2D_rotated()
a = torch.Tensor([[1,2,4,4, 0]]).cuda()
b = torch.Tensor([[1,2,4,4, math.pi]]).cuda()
iou = iou_rotate(a,b)
print(iou)```

And results returned
```
tensor([[1.]], device='cuda:0')
```
I noticed that rotate IOU is not different between angle 0 and PI radians.
I think loss value will depend on this IOU. So how to fit when training so that the angle returns from [0, 2*PI].

**Comment by csuhan** [comment]
`BboxOverlaps2D_rotated` is just used as anchor assignment metric, not IoU loss,
If you want to use IoU Loss, refer to https://github.com/csuhan/s2anet/blob/master/configs/rotated_iou/README.md

Besides, IoU can not reflect orientation (or angle). Instead, the traditional 5 param  (x,y,w,h,a) does.
